Description: build with kf 5.10
 this upload of kio is just to fix some failing tests, remove this patch once
 the rest of kf 5.11 is up
Author: Jonathan Riddell
Forwarded: no
Last-Update: 2015-06-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: kio-5.11.0/CMakeLists.txt
===================================================================
--- kio-5.11.0.orig/CMakeLists.txt
+++ kio-5.11.0/CMakeLists.txt
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 2.8.12)
 project(KIO)
 
 include(FeatureSummary)
-find_package(ECM 5.11.0  NO_MODULE)
+find_package(ECM 5.10.0  NO_MODULE)
 set_package_properties(ECM PROPERTIES TYPE REQUIRED DESCRIPTION "Extra CMake Modules." URL "https://projects.kde.org/projects/kdesupport/extra-cmake-modules")
 feature_summary(WHAT REQUIRED_PACKAGES_NOT_FOUND FATAL_ON_MISSING_REQUIRED_PACKAGES)
 
@@ -22,8 +22,8 @@ include(ECMSetupVersion)
 include(ECMGenerateHeaders)
 include(ECMMarkNonGuiExecutable)
 
-set(KF5_VERSION "5.11.0") # handled by release scripts
-set(KF5_DEP_VERSION "5.11.0") # handled by release scripts
+set(KF5_VERSION "5.10.0") # handled by release scripts
+set(KF5_DEP_VERSION "5.10.0") # handled by release scripts
 
 ecm_setup_version(
   ${KF5_VERSION}
